### ISSUE TITLE
js-norm.c:721 edge-case NULL byte write

### ISSUE THREAD

**Issue by felixgr** [vrtadmin/clamav-devel#38]
In ```handle_df``` if strlen(str) is zero, a NULL byte will be written to the adjacent buffer: ```str[len-1] = '\0';```

Repro.bzip2.base64:

```
QlpoOTFBWSZTWX3YLFQAAk+fjpBAAAUBABAADCBcAAEAABAACAAIIABQoaaYABFFHqDRtTTUyNUl
wAC2KC4q13sZyEKqMGR9CIEBbFBftax+LuSKcKEg+7BYqA==
```

**Comment by steve-morgan** [comment]
Tracking with https://bugzilla.clamav.net/show_bug.cgi?id=11445.

**Comment by felixgr** [comment]
Was there any progress? Can you add groebert at google.com to the ACL of the bugzilla bug?

**Comment by steve-morgan** [comment]
The fix for this issue shipped with ClamAV 0.99.1.
